tests/conftest.py

```diff
@@ -45,6 +52,21 @@
 ERT_RUNPATH = f"_ERT_{FmuEnv.RUNPATH.name}"
 
 
+def pytest_addoption(parser):
+    parser.addoption(
+        "--prod", action="store_true", help="Use schemas/metadata with production URLs."
+    )
```

Comment:
Makes pytest aware of our custom `pytest --prod` command, configured below. But also set at the top of this file because pytest, and some tests, do funky things with environment variables. Since the models in dataio look for this environment variable it is safest to ensure it's set _before_ they are imported, but are still set by the pytest config anyway